### PR TITLE
Added ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by Aman-Codes**
@rpotter12 @xandao6 Please review my pull request and do let me know if any changes are required.
Added bug_report_template.md, feature_request_template.md and pull_request_template.md
Closes: #230 
### Bug report
bug report

### Feature request
feature request

### Pull request
pull request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 24
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031&bid=17118748)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## Issue that this pull request solves
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe):
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 3 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Documentation update (Documentation content changed)
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 3 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 3 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 4 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/pull_request_template.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Documentation update (Documentation content changed)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/pull_request_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 4 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031&bid=17118748)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] Bugfix (non-breaking change which fixes an issue)
+  - [ ] New feature (non-breaking change which adds functionality)
+  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+  - [ ] Documentation update (Documentation content changed)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] My code follows the style guidelines of this project
+  - [ ] I have performed a self-review of my own code
+  - [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/BugReportTemplate.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/BugReportTemplate.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+  - [ ] Documentation update (Documentation content changed)
+  - [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] My code follows the style guidelines of this project
+  - [ ] I have performed a self-review of my own code
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] I have commented my code, particularly in hard-to-understand areas
+  - [ ] I have made corresponding changes to the documentation
+  - [ ] My changes generate no new warnings
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] New feature (non-breaking change which adds functionality)
+  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+  - [ ] Documentation update (Documentation content changed)
+  - [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] My code follows the style guidelines of this project
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] Bugfix (non-breaking change which fixes an issue)
+  - [ ] New feature (non-breaking change which adds functionality)
+  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 23]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] Other (please describe): 
+
+## Checklist
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] My code follows the style guidelines of this project
+  - [ ] I have performed a self-review of my own code
+  - [ ] I have commented my code, particularly in hard-to-understand areas
+  - [ ] I have made corresponding changes to the documentation
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 24]
```diff
@@ -0,0 +1,35 @@
...
+
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] Bugfix (non-breaking change which fixes an issue)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 25]
```diff
@@ -0,0 +1,35 @@
...
+  - [ ] New feature (non-breaking change which adds functionality)
+  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+  - [ ] Documentation update (Documentation content changed)
+  - [ ] Other (please describe):
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/ISSUE_TEMPLATE/PullRequestTemplate.md [hunk 26]
```diff
@@ -0,0 +1,35 @@
...
+## Proposed changes
+
+Brief description of what is fixed or changed
+
+## Types of changes
+
+_Put an `x` in the boxes that apply_
+
+  - [ ] Bugfix (non-breaking change which fixes an issue)
+  - [ ] New feature (non-breaking change which adds functionality)
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/PullRequestTemplate.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5221031)

File: .github/IssueTemplate/PullRequestTemplate.md [hunk 27]
```diff
@@ -0,0 +1,35 @@
+## Issue that this pull request solves
```
Comment (priyansh19) [inline .github/IssueTemplate/PullRequestTemplate.md]: Hey @Aman-Codes,
Your PR looks good but solve only one issue at a time
you have solved issue #233 and #230 both

**Comment by xandao6** [comment]
Yes, create different PR's, will be good for you also! You will earn more GSSOC points.

**Pull request closed without merging**